systems/billing/api-gateway/pkg/client/invoice_test.go

```diff
@@ -34,7 +33,7 @@ func TestBillingClient_AddInvoice(t *testing.T) {
 	"number": "LAG-1234-001-002",
 	"issuing_date": "2022-04-30",
 	"status": "finalized",
-	"payment_status": "succeeded",
+	"invoice_status": "succeeded",
```

Comment:
What does `invoice status` entail? If it's marked as "succeeded," I assume it should indicate payment status instead. If referring specifically to `invoiceStatus`, it should only be categorized as `paid` or `pending`.